knora/dsplib/utils/stack_handling.py

```diff
@@ -0,0 +1,52 @@
+import subprocess
+import time
+import re
+
+import requests
+
+
+def start_stack() -> None:
+    """
+    Start the Docker containers of DSP-API, and load some basic data models and data.
+    """
+    # start up the fuseki database
+    completed_process = subprocess.run("docker compose up db -d", shell=True)
+    if not completed_process or completed_process.returncode != 0:
+        exit(1)
+    time.sleep(5)
+
+    # create the "knora-test" repository
+    url_prefix = "https://raw.githubusercontent.com/dasch-swiss/dsp-api/main"
+    repo_template_response = requests.get(f"{url_prefix}/webapi/scripts/fuseki-repository-config.ttl.template")
+    if not repo_template_response or repo_template_response.status_code != 200:
+        exit(1)
+    repo_template = re.sub(r"@REPOSITORY@", "knora-test", repo_template_response.text)
+    response = requests.post(url="http://0.0.0.0:3030/$/datasets", files={"file": ("file.ttl", repo_template)},
```

Comment:
@seakayone Here I am stuck. I get a 500 status and the message `[line: 1, col: 1 ] Out of place: [MINUS]`. 
This line tries to imitate https://github.com/dasch-swiss/dsp-api/blob/main/webapi/scripts/fuseki-functions.sh#L70
Like in line 41 below, I try to imitate that a file is sent instead of a string, as documented here: https://requests.readthedocs.io/en/latest/user/quickstart/#post-a-multipart-encoded-file.

Balduin already tried to help me, but we both don't know what the problem is. Could you help out?